cmd/rook/rook/rook.go

```diff
@@ -76,9 +75,6 @@ func SetLogLevel() {
 }
 
 func LogStartupInfo(cmdFlags *pflag.FlagSet) {
-	// workaround a k8s logging issue: https://github.com/kubernetes/kubernetes/issues/17162
-	flag.CommandLine.Parse([]string{})
```

Comment:
is `SetLoggingFlags()` called everywhere that `LogStartupInfo()` was called? If not, we might see the k8s logging issue mentioned in that comment.

Reply:
yes, it is now being called from all the backend `init()` calls.